src/scenes/Event/store.js

```diff
@@ -19,7 +19,12 @@ const INIT = {
   loading: true,
   oracles: [],
   topics: [],
+  leaderboardVotes: [],
   votes: [],
+  bets: [],
+  resultsets: [],
+  finalizes: [],
+  withdraws: [],
```

Comment:
i'm thinking we can just use one single array (`transactionHistoryItems`) for all: `transactions, bets, votes, resultSets, finalizes, withdraws` unless you have a good reason to keep them separated? and we just sort the entire list based on shared fields e.g. `blockTime`.

Reply:
oh looks like `this.transactions` is what i described. nit: to rename to `transactionHistoryItems` to be clear

Reply:
Absolutely sure. The only reason I made these arrays is in case in the future we need to use any specific one of them. It's my personal coding habit, not sure if it's a good one tho...

Reply:
Ya just use the single array for now. If we have a use for using individual ones later it will be a quick change to track each one.